README.md

```diff
@@ -56,7 +58,7 @@ The best way to start the app and work on it is by using `npm run dev`, which wi
 After the app starts, it will be accessible on `localhost:3000` (unless the port was modified via `.env`).
 
 ## Handling App Events Locally
-In order to make your app accessible to platforms like Slack, you'll need to use an app like [`ngrok`](https://github.com/bubenshchykov/ngrok) to obtain a URL which can be used within your Slack app configuration. After running `npm i` to install dependencies, simply `npm run tunnel` to create the tunnel, then use the following URL output to route web traffic to your app.
+In order to make your app accessible to platforms like Slack, you'll need to use an app like [`ngrok`](https://ngrok.com/) to obtain a URL which can be used within your Slack app configuration. After installing ngrok, simply run `ngrok http 3000` to create the tunnel, then use the following URL output to route web traffic to your app.
```

Comment:
```suggestion
In order to make your app accessible to platforms like Slack, you'll need to use an app like [`ngrok`](https://ngrok.com/) to obtain a URL which can be used within your Slack app configuration. Install ngrok globally with `npm i ngrok -g` and then run `ngrok http 3000` to create the tunnel. After it starts, use the following URL output to route web traffic to your app.
```

Reply:
I'd rather not install ngrok with npm, because that will hit the same problems that Allyson ran into on Windows computers not setup to install addons. If they want to they can, but there are better install methods [here](https://ngrok.com/download).